build.gradle

```diff
@@ -142,19 +145,55 @@ tasks.withType(JavaCompile).configureEach {
 	options.compilerArgs.add '-XDstringConcat=inline'
 }
 
-build.dependsOn('installLocalGitHookPostCheckout')
-tasks.register('installLocalGitHookPostCheckout', Copy) {
-	group = 'git'
-	from new File(rootProject.rootDir, '.scripts/post-checkout')
-	into { new File(rootProject.rootDir, '.git/hooks') }
-	filePermissions {
-		unix(0775)
+
+// Setup local git hooks
+
+def ensureSymlink(link, target) {
```

Comment:
```suggestion
def ensureSymbolicLink(link, target) {
```

Reply:
Again, Symlink is the much more common name for symbolic links. It's like calling tabs 'tabulations'.